cmd/antrea-agent/options.go

```diff
@@ -225,6 +233,28 @@ func (o *Options) validateFlowExporterConfig() error {
 			}
 			o.pollInterval = flowPollInterval
 		}
+		// Parse the given activeFlowExportTimeout config
+		if o.config.ActiveFlowExportTimeout != "" {
+			o.activeFlowTimeout, err = time.ParseDuration(o.config.ActiveFlowExportTimeout)
+			if err != nil {
+				return fmt.Errorf("ActiveFlowExportTimeout is not provided in right format")
+			}
+			if o.activeFlowTimeout < o.pollInterval {
+				o.activeFlowTimeout = o.pollInterval
+				klog.Warningf("ActiveFlowExportTimeout is recommended to be greater than or equal to FlowPollInterval")
```

Comment:
not "is recommended", it should be "must be"
you are effectively preventing the user from using an ActiveFlowExportTimeout less than FlowPollInterval
same below